orcid-persistence/src/main/java/org/orcid/persistence/messaging/JmsMessageSender.java

```diff
@@ -98,7 +95,9 @@ public boolean send(LastModifiedMessage mess, JmsDestination d){
         try{
             return this.sendMap(mess.getMap(), d);                             
         }catch(JmsException e){
-            flagConnectionProblem(e);
+            //TODO: How we unflag the problem?
```

Comment:
The problem is unflagged by the scheduled code further down the class.  Spring handles the timer.

/** retry connecting if bad after every couple of minutes
     * 
     */
    @Scheduled(fixedDelay=60000)
    public void timer(){
        synchronized(this){
            if (pauseForAWhile){
                pauseForAWhile = false;
            }            
        }
    }